generator/generator_tool/generator_tool.cpp

```diff
@@ -387,23 +376,13 @@ MAIN_WITH_ERROR_HANDLING([](int argc, char ** argv)
 
       if (!FLAGS_uk_postcodes_dataset.empty() || !FLAGS_us_postcodes_dataset.empty())
       {
-        if (!countryParentGetter)
-        {
-          LOG(LCRITICAL,
-              ("Countries file is needed. Please set countries file name (countries.txt). "
-               "File must be located in data directory."));
-          return EXIT_FAILURE;
-        }
-
-        auto const topmostCountry = (*countryParentGetter)(country);
         bool res = true;
-        if (topmostCountry == "United Kingdom" && !FLAGS_uk_postcodes_dataset.empty())
+        if (!FLAGS_uk_postcodes_dataset.empty() && strings::StartsWith(country, "UK_"))
```

Comment:
Do you have any idea on how to pass any other external databases with postal codes in the same format without introducing a command-line flag for each country? E.g. if I want to add swiss postal codes? Maybe simply pass a folder with files starting with prefixes matching our countries?